themes/godotengine/layouts/features.htm

```diff
@@ -73,19 +73,19 @@ <h2>innovative design</h2>
   <img src="{{ 'assets/features/3dgames.jpg' | theme }}" alt="">
 
   <div class="title-padded base-padding">
-    <h2>One of the best looking 3D engines out there</h2>
+    <h2>Gorgeous 3D Graphics</h2>
     <em>
-      The whole new physically based renderer comes with a bunch features that will make your games look incredible.
+      The brand new physically based renderer comes with a ton of features that will make your games look incredible.
     </em>
   </div>
   <div class="flex eqsize responsive base-padding">
     <ul>
-      <li><b>Innovative architecture</b>, that combines the best of forward rendering with the efficiency of deferred rendering.</li>
+      <li><b>Innovative architecture</b> that combines the best of forward rendering with the efficiency of deferred rendering.</li>
       <li><b>Physically-based rendering</b> with full MSAA support</li>
       <li><b>Full principled BSDF</b> with Subsurface Scattering, reflection, refraction, anisotropy, clearcoat, transmittance and more.</li>
     </ul>
     <ul>
-      <li><b>Global illumination</b> real-time for gorgeous graphics and baked for good results even on low end devices.</li>
+      <li><b>Global illumination</b> for real-time gorgeous graphics. It can also be pre-baked for beautiful results even on low end devices.</li>
```

Comment:
Here the sentence actually refers to two features: Global Illumination, which is real time, and Lightmapping, with needs baking but is a lot faster on low end devices. Not sure how to best phrase it though, @reduz?

Reply:
Maybe it could be split into two list items, one for GI and one for Lightmapping? Something like:
"Global illumination for rendering realistic environments in real time"
"Lightmapping for pre-baked, high-performance lighting that looks great even on low-end devices"